api/session/auth_session.py

```diff
@@ -91,12 +92,28 @@ def clear_session():
         session.clear()
         clear_sentry()
 
+        redirect_route = (  # TODO: Remove defaulting to Magic Links, instead use return_app
+            "magic_links_bp.signed_out"
+        )
+        if return_app := request.args.get("return_app"):
```

Comment:
Could we update the openapi schema to reflect this optional argument?

https://github.com/communitiesuk/funding-service-design-authenticator/blob/eb62551669cc80a0cae6672c37111c7eea905f7c/openapi/api.yml#L136-L145

i.e.

```yaml
/sessions/sign-out:
  get:
    tags:
      - sessions
    summary: Signs out a user
    description: Signs out a user who has authenticated via a magic link
    operationId: api.AuthSessionView.clear_session
    parameters:
      - name: return_app
        in: query
        description: Optional parameter to specify the return app
        required: false
        schema:
          type: string
    responses:
      302:
        description: SUCCESS - Active user session cleared and redirected to the signed out page
```

Reply:
yeah good shout!

Reply:
Done this, also aded it for` sso/login`